From: Markus Elfring <elfring@users.sourceforge.net>
Date: Sun, 20 Aug 2017 20:08:25 +0200
Subject: ceph: delete an unnecessary return statement in update_dentry_lease()
Git-commit: b529d1b382f77ec6221f9c5fffb17939e92629e4
Patch-mainline: v4.14-rc1
References: FATE#324714

The script "checkpatch.pl" pointed information out like the following.

WARNING: void function return statements are not generally useful

Thus remove such a statement in the affected function.

Signed-off-by: Markus Elfring <elfring@users.sourceforge.net>
Reviewed-by: Yan, Zheng <zyan@redhat.com>
Signed-off-by: Ilya Dryomov <idryomov@gmail.com>
Acked-by: Luis Henriques <lhenriques@suse.com>
---
 fs/ceph/inode.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/fs/ceph/inode.c b/fs/ceph/inode.c
index 087d9ea3a153..277c7cd7b1ab 100644
--- a/fs/ceph/inode.c
+++ b/fs/ceph/inode.c
@@ -1067,7 +1067,6 @@ static void update_dentry_lease(struct dentry *dentry,
 	spin_unlock(&dentry->d_lock);
 	if (old_lease_session)
 		ceph_put_mds_session(old_lease_session);
-	return;
 }
 
 /*

